cmd/atlas/internal/cmdapi/migrate.go

```diff
@@ -50,6 +58,12 @@ var (
 		RevisionSchema string
 		Force          bool
 		Verbose        bool
+		Lint           struct {
+			Format  string // log formatting.
```

Comment:
This comments are no sentences, therefore I don't like they include interpunctuation (personal pref though)